### PR TITLE
Removed controller support. Added player slowdown with lshift

### PR DESCRIPTION
**Pull request by shraymittal**

### CODE REVIEW

**Comment by shraymittal** [comment]
the force push and the merge conflicts were caused by my last PR having commits that for some reason didn't register as authored by me. I fixed that, no code changes

File: BismarkUnchained/scenes/Player/Player.gd [hunk 1]
```diff
@@ -21,42 +19,26 @@ func damage():
...
+	if Input.is_action_pressed("player_left"):
+		velocity += Vector2(-1, 0)
+	if Input.is_action_pressed("player_right"):
+		velocity += Vector2(1, 0)
 	
 	#Finding difference between current and target velocity, and adjusting for acceleration
-	velocity = velocity * max_velocity
+	velocity = velocity.normalized() * max_velocity
+	if Input.is_action_pressed("player_slowdown"):
+		velocity = velocity.normalized() * slow_velocity
```
Comment (Noviv) [inline BismarkUnchained/scenes/Player/Player.gd:35]: not sure if this affects the math, but you're normalizing twice when it makes sense to compute the velocity scalar (`max_velocity` or `max_velocity * slow_velocity`) then multiply by the `velocity.normalized()`

File: BismarkUnchained/scenes/Player/Player.gd [hunk 2]
```diff
@@ -65,16 +47,9 @@ func move_translate(delta):
...
-		if direction.length() > 0.5:
-			last_dir = Vector2(direction.x, direction.y)
-			rotate_dir(direction)
-	else:
-		var direction = get_global_mouse_position() - get_global_position()
-		last_dir = Vector2(direction.x, direction.y)
-		rotate_dir(direction)
-		
+	var direction = get_global_mouse_position() - get_global_position()
+	last_dir = Vector2(direction.x, direction.y)
```
Comment (Noviv) [inline BismarkUnchained/scenes/Player/Player.gd:51]: `last_dir = direction`? not sure if that can be used

**Review by Noviv — APPROVED** [review]
lgtm, but left a couple comments

**Pull request merged**